### ISSUE TITLE
Flash Player Debug versions

### ISSUE THREAD

**Issue by r3sus** [CleanFlash/installer#13]
Hello and thanks again for making this! 

There are debug version of plugins and sa available: 
https://www.flash.cn/support/debug-downloads

That would be useful to add them in installer and provide options on installing. The more flexible solution would be installing both and make switch based on simple file renaming. Or at least address their existence in readme. 

Otherwise, I think flashpatch is still applicable? 

What do you think?

Thanks.

**Comment by DraconicNEO** [comment]
I think this would be a good idea. Although I'm not sure if [Flashpatch](https://github.com/darktohka/FlashPatch) supports the Debug versions yet. 

@darktohka Can you tell us if Flashpatch supports the Debug version of Flash.cn?

**Comment by darktohka** [comment]
Yes, FlashPatch does support the debug versions. I'm considering releasing a separate version of the installer just for the debug variant, once I get around to make the new installer for 34.0.0.175.

**Comment by r3sus** [comment]
I'm up to flexible solution, when you can switch between debug and normal versions just by renaming files with i.e. batch script. Altho it might require to exit all applications using Flash. Should be better then reinstalling flash every time. 
I think many flash applications has non-critical errors. So if you just want to play game or test your application for end-user, you don't need any error messages, which might be irrigating. In case you want to debug, you'd just switch to it. 
However, I don't sure if people work like this. Correct me if I'm wrong. Personally, I don't use browser for flash, only projectors, players. 
Thanks.

**Comment by darktohka** [comment]
I'd rather not ship the debug versions together with the actual installer, because it'd double the installer file size (which is already 30 megabytes). That being said, I've just released version 34.0.0.175 with a separate installer that installs the debug versions of Flash. I'm not entirely sure how people work with these debug versions, but this should be enough to get things started.

If anyone has more information on how typical workflows look like when using debug versions, I'd be interested in hearing about that.

**Comment by r3sus** [comment]
Fair, thanks.
